app/util/jira/populate_db.sh

```diff
@@ -24,17 +46,33 @@ JIRA_DB_PASS="Password1!"
 
 # Jira version variables
 SUPPORTED_JIRA_VERSIONS=(8.0.3 7.13.15 8.5.8)
+
+# JSD section
+if [[ ${jsd} == 1 ]]; then
+  JIRA_CURRENT_DIR="/opt/atlassian/jira-servicedesk/current"
+  JIRA_SETENV_FILE="${JIRA_CURRENT_DIR}/bin/setenv.sh"
+  JIRA_VERSION_FILE="/media/atl/jira/shared/jira-servicedesk.version"
+  SUPPORTED_JIRA_VERSIONS=(4.13.0)
+fi
+
 JIRA_VERSION=$(sudo su jira -c "cat ${JIRA_VERSION_FILE}")
 if [[ -z "$JIRA_VERSION" ]]; then
-  echo The $JIRA_VERSION_FILE file does not exists or emtpy. Please check if JIRA_VERSION_FILE variable \
+  echo The ${JIRA_VERSION_FILE} file does not exists or emtpy. Please check if JIRA_VERSION_FILE variable \
     has a valid file path of the Jira version file or set your Cluster JIRA_VERSION explicitly.
   exit 1
 fi
 echo "Jira Version: ${JIRA_VERSION}"
 
 # Datasets AWS bucket and db dump name
 DATASETS_AWS_BUCKET="https://centaurus-datasets.s3.amazonaws.com/jira"
+if [[ ${jsd} == 1 ]]; then
+  DATASETS_AWS_BUCKET="https://centaurus-datasets.s3.amazonaws.com/jsd"
+fi
 DATASETS_SIZE="large"
+if [[ ${jsd} == 1 && ${small} == 1 ]]; then
+  # Only JSD supports "small" dataset
+  DATASETS_SIZE="small"
+fi
```

Comment:
How do you think, maybe it will  be nice to merge "if" statements, something like this?
DATASETS_SIZE="large"
if [[ ${jsd} == 1 ]]; then
  DATASETS_AWS_BUCKET="https://centaurus-datasets.s3.amazonaws.com/jsd"
  if [[ ${small} == 1 ]]; then
    # Only JSD supports "small" dataset
    DATASETS_SIZE="small"
  fi
fi